bluetooth/README.md

```diff
@@ -37,4 +37,15 @@
 ||/close|||Close bluetooth. Any connection will be break and radio will no longer transimitting.|
 ||/discovery|||Start discovery around other bluetooth device.|
 
+## Bluetooth state emit
 
+* flora subscribe name: `'yodaos.apps.bluetooth.status'`
+
+* flora message format: `[ code ]`
+
+|status|code|
+|---|---|
+|关闭|0|
+|打开|1|
+|断开|2|
+|连接|3|
```

Comment:
Documentation should be in English by default :)

Reply:
I use Chinese is because customers interface document describe it by Chinese. 
We keep it consistent can avoid understanding trouble. 
But any way, I modified it. Due to these words is simple, should not bring us understanding trouble.

Reply:
The bluetooth README should owned by ourselves :)